projects/docs/docs/manifests/config.md

```diff
@@ -1,82 +0,0 @@
----
-title: Config.swift
-slug: '/manifests/config'
-description: This page documents how to use the Config manifest file to configure Tuist's functionalities globally.
```

Comment:
I would still keep the high-level explainer here and use docc for the detail documentation. I'm a little bit worried about users being disoriented by having to switch between two sites for docs and therefore, would try to keep that at minimum.

Users should also still be able to search for `Config.swift` and get an idea for that does (this applies to other manifests, too)

Reply:
Good point about the search, maybe we can add a super brief recap of the various manifests with the corresponding link in the `api-reference.md`

Reply:
Fair point. Any suggestion on how the overview + the link would look like? @fortmarek @danyf90

Reply:
I think it should be something like:
```
## Manifests

Tuist supports a number of manifests to configure various aspects of project generation. This document provides a recap of them and a link to their API documentation for more details

### Config.swift

it configures bla bla bla

For more details, check the [API reference](link)

### Project.swift

…

## API reference

<what you have now in the api-reference.md>
```

Reply:
@fortmarek what do you think about the above?

Reply:
Sorry for the late response, I was sick 🤕 

I'd still consider adding small examples of those manifests but otherwise, looks good 👍

Reply:
I would avoid having the example, to avoid the risk of them becoming obsolete

Reply:
if the examples are more-or-less contrived, I don't think it's a big possibility - they only might break during major releases. I'd run with this risk - I think it's good to give people more high-level idea of what the manifests do.

Reply:
@fortmarek and @danyf90, I updated the PR, does this looks like a good compromise? :)

Reply:
GTM